borsh/src/de/mod.rs

```diff
@@ -1,3 +1,7 @@
+use std::str::FromStr;
+
+use bigdecimal::BigDecimal;
```

Comment:
This import, and all other decimal-specific code should be gated by cfg (see how Rc is implemented).

it might be even better to move this to a module and gate just the module, though that goes against Rc precedent.